models/iris_gps_height/iris_gps_height.sdf

```diff
@@ -0,0 +1,618 @@
+<sdf version='1.7'>
```

Comment:
Is there anything different compared to the default iris? If there is nothing different, you can also just reference to default iris as done in https://github.com/PX4/sitl_gazebo/tree/master/models/iris_opt_flow